src/distributions/utils.rs

```diff
@@ -186,8 +185,6 @@ mod simd_wmul {
         };
     }
 
-    #[cfg(target_feature = "sse2")]
-    wmul_impl_16! { u16x4, __m64, _mm_mulhi_pu16, _mm_mullo_pi16 }
     #[cfg(target_feature = "sse4.2")]
```

Comment:
It seems like this could be relaxed to be `"sse2"`, as that's when `_mm_mulhi_epu16`/`_mm_mulhi_epu16` are available.